### DISCUSSION TITLE
Dependency updates missed for (sometimes) many months

### DISCUSSION THREAD

**Discussion by dogmatic69** [renovatebot/renovate#9755] [Suggest an Idea]
**What Renovate type, platform and version are you using?**

Hosted app on GitHub

**Describe the bug**

Not all versions are updated, sometimes weeks or months pass with no updates in dependencies

**Relevant debug logs**

Not sure how to see logs for things that are not updated

**Have you created a minimal reproduction repository?**

- [ ] I have provided a minimal reproduction repository
- [x] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**
Here are updates to @types/node

![image](https://user-images.githubusercontent.com/94674/116305350-f2350180-a7a3-11eb-9c99-d0239b92077c.png)

But renovate only updated [5 days ago](https://github.com/renovatebot/renovate/pull/9694)

My expectaction would be that all releases trigger an update, especially when it's been days

**Comment by rarkins** [comment]
Renovate updated that particular dependency 5 days ago because we apply a stabilityDays=7 setting on this repository. The example you've given isn't valid so I've moved this to a discussion.
